### PR TITLE
rm links to auth samples /2

### LINKED ISSUE #32262: Remove links to auth samples
See https://github.com/dotnet/aspnetcore/issues/53700

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/security/authentication/samples?view=aspnetcore-8.0

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authentication/samples.md

### Document ID

80b7f7cf-dc2e-137a-5aa7-fa88da63ac80

### Article author

@Rick-Anderson

---
[Associated WorkItem - 240942](https://dev.azure.com/msft-skilling/Content/_workitems/edit/240942)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #32262
Fixes https://github.com/dotnet/aspnetcore/issues/53700

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/grpc/authn-and-authz.md](https://github.com/dotnet/AspNetCore.Docs/blob/6c4a2d7200fee4bbea25aaa05bdff5b7496dd229/aspnetcore/grpc/authn-and-authz.md) | [Authentication and authorization in gRPC for ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/grpc/authn-and-authz?branch=pr-en-us-32263) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  can probably have more elegant wording.  cc @guardrex

File: aspnetcore/security/authentication/samples.md [hunk 1]
```diff
@@ -10,22 +10,4 @@ uid: security/authentication/samples
...
-
-## Obtain and run the samples
-
-The sample links provided in this article provide samples for the upcoming release of ASP.NET Core. To obtain a sample for the current release or a prior release, perform the following steps:
-
-* Select the release branch of the [ASP.NET Core repository](https://github.com/dotnet/aspnetcore). For example, the `release/5.0` branch contains the samples for the ASP.NET Core 5.0 release.
-* Clone or download the ASP.NET Core repository.
-* On your local system, verify installation of the [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core) version matching the clone of the ASP.NET Core repository.
-* Navigate to a sample in `aspnetcore/src/Security/samples` folder and run the sample with the [`dotnet run` command](/dotnet/core/tools/dotnet-run).
+See  <xref:security/authentication/index> and the topics in the under Authentication for and overview of authentication and code snippets.
```
Comment (guardrex) [inline aspnetcore/security/authentication/samples.md:13]: NIT stray space and perhaps bold for the ToC heading ...

```suggestion
See <xref:security/authentication/index> and the topics in the under **Authentication** for and overview of authentication and code snippets.
```

File: aspnetcore/security/authentication/samples.md [hunk 2]
```diff
@@ -10,22 +10,4 @@ uid: security/authentication/samples
...
-
-## Obtain and run the samples
-
-The sample links provided in this article provide samples for the upcoming release of ASP.NET Core. To obtain a sample for the current release or a prior release, perform the following steps:
-
-* Select the release branch of the [ASP.NET Core repository](https://github.com/dotnet/aspnetcore). For example, the `release/5.0` branch contains the samples for the ASP.NET Core 5.0 release.
-* Clone or download the ASP.NET Core repository.
-* On your local system, verify installation of the [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core) version matching the clone of the ASP.NET Core repository.
-* Navigate to a sample in `aspnetcore/src/Security/samples` folder and run the sample with the [`dotnet run` command](/dotnet/core/tools/dotnet-run).
+See <xref:security/authentication/index> and the topics in the under **Authentication** for an overview of authentication and code snippets.
```
Comment (tdykstra) [inline aspnetcore/security/authentication/samples.md:13]:
```suggestion
See <xref:security/authentication/index> and the articles in the table of contents under **Authentication** for an overview of authentication and code snippets.
```

I'm guessing the TOC is what you had in mind.

File: aspnetcore/security/authentication/samples.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Authentication samples for ASP.NET Core
+title: Authentication links ASP.NET Core
 author: rick-anderson
-description: Provides links to the authentication samples in the ASP.NET Core repository.
+description: Provides links to the authentication topics in the ASP.NET Core repository.
```
Comment (tdykstra) [inline aspnetcore/security/authentication/samples.md:4]:
```suggestion
description: Provides links to the authentication articles in the ASP.NET Core repository.
```

**Review by tdykstra — APPROVED** [review]
A couple suggestions.

File: aspnetcore/security/authentication/samples.md
Comment (tdykstra) [inline aspnetcore/security/authentication/samples.md:1]: This page no longer seems useful, I think we would do better to delete it, and redirect to the overview.

**Review by tdykstra — APPROVED** [review]

File: aspnetcore/security/authentication/samples.md
Comment (Rick-Anderson) [reply aspnetcore/security/authentication/samples.md:1]: I knew you'd know what to do. thanks

File: aspnetcore/toc.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-items:
+items:
```
Comment (Rick-Anderson) [inline aspnetcore/toc.yml:1]:
```suggestion
items:
```

**Pull request merged**